config/static.c

```diff
@@ -26,103 +35,121 @@
 #include "gatekeeper_lls.h"
 #include "gatekeeper_rt.h"
 
+#define RTE_LOGTYPE_CONFIG RTE_LOGTYPE_USER2
+
+/* TODO Get the install-path via Makefile. */
+#define LUA_BASE_DIR "./lua/"
+#define LUA_CONFIG_FILE     "gatekeeper_config.lua"
+
+struct gatekeeper_config {
+	struct bp_config 	*bp_conf;
+	struct catcher_config 	*catcher_conf;
+	struct dynamic_config 	*dy_conf;
+	struct cps_config 	*cps_conf;
+	struct ggu_config 	*ggu_conf;
+	struct gk_config 	*gk_conf;
+	struct gt_config 	*gt_conf;
+	struct rt_config 	*rt_conf;
+};
+
+static lua_State *lua_state;
```

Comment:
For mailboxes, that makes sense. For the exiting process, a concern is that this process needs to remember which functional blocks are launched before, so it can call the corresponding destroying functions. The current code doesn't seem to do so. If we can keep these information in the C host program, we can omit the ``gatekeeper_destroy()`` function. Since keeping the Lua state throughout the program is not desired. To keep information about which block is launched, a simple way is to keep an "uint64_t mark", each bit of this variable represents a functional block. Is this ok?